test/k8s/manifests/l7-policy.yaml

```diff
@@ -1,21 +0,0 @@
-apiVersion: "cilium.io/v2"
```

Comment:
This policy still seems to be used here: https://github.com/cilium/cilium/blob/062549799357504bb13ceb1e7854aef56ecf6130/test/k8s/updates.go#L63

Reply:
thanks thanks, I will wait for test-1.25-net-next to correct all mistakes :100: